jake/audit/audit.py

```diff
@@ -13,11 +13,15 @@
 # limitations under the License.
 import logging
 
+from typing import List
+
+from jake.types.coordinateresults import CoordinateResults
+
 class Audit(object):
     def __init__(self):
         self._log = logging.getLogger('jake')
 
-    def auditResults(self, results):
+    def auditResults(self, results: List[CoordinateResults]):
```

Comment:
My understanding is this typing annotation, without associated `is instance()`, does nothing yet...but I don't see a need to remove it yet. We should just not be lulled into thinking anything will be type checked for us.

Reply:
That is my understanding as well. I have no idea why you can even do this and have it not throw a `TypeError` or something akin, but I bet it's mainly for IDEs so you can see the signature of it when developing.